internal/lib/lib.go

```diff
@@ -461,6 +453,12 @@ func GetVipNetworkList() ([]string, error) {
 	for _, subnet := range vipNetworkListObj {
 		vipNetworkList = append(vipNetworkList, subnet.NetworkName)
 	}
+	if len(vipNetworkList) > 1 {
+		// Only AWS cloud supports multiple VIP networks
+		if !(IsPublicCloud() && GetCloudType() != CLOUD_AWS) {
```

Comment:
We can validate for only AWS

Reply:
Thats what we are checking in the if condition. If the cloud type is AWS return error.